python/GafferUI/ColorChooser.py

```diff
@@ -159,14 +159,20 @@ def __init__( self, color=imath.Color3f( 1 ), **kw ) :
 		self.__sliders = {}
 		self.__numericWidgets = {}
 		self.__channelLabels = {}
+		self.__channelSpacers = {}
 		self.__componentValueChangedConnections = []
 
 		with self.__column :
 
 			with GafferUI.GridContainer( spacing = 4 ) :
 
 				# sliders and numeric widgets
-				for row, component in enumerate( "rgbahsv" ) :
+				components = "rgb_a_hsv_tmi"
+				for row, component in enumerate( components ) :
+					if component == "_" :
+						self.__channelSpacers[components[row - 1]] = GafferUI.Spacer( imath.V2i( 0, 4 ), imath.V2i( 0, 4 ), parenting = { "index" : ( 0, row ) } )
```

Comment:
I know we're going to drop this before merging, but here's an observation to feed into the layout design : it does feel potentially useful to provide some grouping in some way, but the spacing feels out of balance with the surrounding widgets when embedded in the NodeEditor, and with the swatches when shown in a floating editor. The slider groups are more closely related to each other than the widgets around them, yet they have bigger spacing.

Reply:
Good point on the spacing, I see what you mean. I'll keep that in mind when we explore the layout options.